### PR TITLE
12 code edits

### PR DESCRIPTION
**Pull request by restrellado**
I tested lines 87-103 with and without the `has_mention` variable and got the same result so I removed it. 

Before calling `unnest`, there are no NAs in the `all_mentions` column when it's a list column. After calling `unnest`, it looks like R drops any row that didn't have a username in the `all_mentions` column. 

@jrosen48 Let's keep this one open until I review the rest of the code, but wanted you to see it and give me a gut check on what I was seeing.

### CODE REVIEW

File: 12-wt-social-network-analysis.Rmd [hunk 1]
```diff
@@ -100,7 +99,6 @@ Let's put these into their own data frame, called `mentions`.
 ```{r}
 mentions <-
   tt_tweets %>%
-  filter(has_mention) %>%
```
Comment (restrellado) [inline 12-wt-social-network-analysis.Rmd]: I got the same `mentions` dataframe before and after removing `has_mention`

Comment (jrosen48) [reply 12-wt-social-network-analysis.Rmd]: ahh good catch. took a look - this looks like it's a product of `unnest()`, which only retains rows that are not length 0 (the rows with `character(0)` - those without any mentions - are automatically removed). in short, good catch, thanks for removing this.

**Comment by restrellado** [comment]
> minor, line 119, add a space in between all_mentions and the = symbol?
> tried to checkout the PR and push it but... oye, ran into trouble.

Yep no problem I'll take care of it!

**Comment by restrellado** [comment]
@jrosen48 this code is around line 163:

```
g %>% 
  mutate(centrality = centrality_authority()) %>% 
  ggraph() +
  geom_node_point(aes(size = centrality, color = centrality)) +
  scale_color_continuous(guide = 'legend') + 
  geom_node_text(aes(label = name)) +
  geom_edge_link() +
  theme_graph()
```

I see size and color mapped to centrality in the legend, but I don't see size or color changing at all in the plot. Was that intended or is this something we need to fix?

File: 12-wt-social-network-analysis.Rmd [hunk 2]
```diff
@@ -118,7 +116,9 @@ What needs to happen to these to make them easier to work with in an edgelist? O
 ```{r}
 edgelist <- 
   mentions %>% 
-  mutate(all_mentions= str_sub(all_mentions, start = 2)) %>% 
+  # remove "@" from all_mentions column
+  mutate(all_mentions = str_sub(all_mentions, start = 2)) %>% 
+  # rename all_mentions to receiver
```
Comment (jrosen48) [inline 12-wt-social-network-analysis.Rmd]: thanks for adding these comments

**Comment by jrosen48** [comment]
Sorry for delay, done.

**Pull request merged**